gcp-observability/build.gradle

```diff
@@ -34,8 +34,7 @@ dependencies {
             libraries.autovalue_annotation,
             libraries.perfmark,
             libraries.opencensus_contrib_grpc_metrics,
-            ('com.google.guava:guava:31.0.1-jre'),
-            ('com.google.errorprone:error_prone_annotations:2.11.0'),
```

Comment:
Was this deletion of `errorprone` intended?

Reply:
Yes. That was previously bumping the version that grpc used elsewhere. It seemed to address a requireUpperBoundDepsMatch failure. With the upgrade it is no longer necessary.